src/components/Reply/Content/index.tsx

```diff
@@ -0,0 +1,85 @@
+import { TextButton } from 'components/Button';
+import { gray500, gray800 } from 'core/Colors';
+import { body2 } from 'core/TextStyles';
+import React, { FC, useCallback, useEffect, useState } from 'react';
+import styled, { css } from 'styled-components';
+
+interface Props {
+  useLineClamp: boolean;
+  maxLine: number;
+  readMoreText: string;
+  hideText: string;
+  onClick?: React.MouseEventHandler<HTMLDivElement>;
+}
+
+export const ReplyContent: FC<Props> = React.memo(props => {
+  const { children, useLineClamp, maxLine, readMoreText, hideText, onClick } = props;
+
+  const [lineClamped, setLineClamped] = useState(true);
+  const [lineClampable, setLineClampable] = useState(false);
+
+  const contentRef = React.useRef<HTMLDivElement | null>(null);
+
+  const getContentClamped = useCallback(() => {
+    const { current } = contentRef;
+
+    if (children && current) {
+      return current.scrollHeight > current.clientHeight;
+    }
+    return false;
+  }, [contentRef, children]);
```

Comment:
```suggestion
  }, [children]);
```
ref는 항상 최신 값을 가지고 있어서 deps에 안들어가도 될듯!

Reply:
확인 후 둘 다 수정했습니당